lib/gl_funcs.c

```diff
@@ -69,21 +69,21 @@ HMODULE ktxOpenGLModuleHandle;
 #define GetOpenGLModuleHandle(flags) dlopen(NULL, flags)
 #define LoadProcAddr dlsym
 #define LIBRARY_NAME NULL
-void* ktxOpenGLModuleHandle;
+void *ktxOpenGLModuleHandle;
```

Comment:
I saw the opposite change being made in basisu_c_binding.{cpp,h}, "foo *bar" became "foo* bar". Why the difference here? Is it a c vs C thing? I prefer "foo* bar" because to my mind the type of "bar" is "pointer to foo".

Reply:
No, this is one of those configurable things. We can go either way. The reason why this style is the default in the Google style because technically the "*" belongs to the variable name (imagine when multiple variables are declared in a comma separated list like `int *a_ptr, b_value, *c_ptr;`).

Reply:
My question here is why the difference in `basisu_c_binding.*` vs within the libktx code?